src/main/java/org/terasology/math/delaunay/Line2f.java

```diff
@@ -0,0 +1,265 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.math.delaunay;
+
+import org.joml.Vector2f;
+import org.joml.Vector2fc;
+import org.terasology.joml.geom.Rectanglef;
+import org.terasology.math.geom.BaseRect;
+
+public class Line2f {
```

Comment:
@Cervator I would not throw this into `joml-ext` right away, but only after careful consideration (let's try to keep that lib high quality instead of dumping things in there). 

We already have [LineSegmentf](https://github.com/MovingBlocks/joml-ext/blob/main/joml-geometry/src/main/java/org/terasology/joml/geom/LineSegmentf.java) which is the same thing in 3D, missing some utility functions.

I'll create a ticket on https://github.com/MovingBlocks/joml-ext to remind us about merging the feature set of this class and the `LineSegment` class we adopted from JOML, and probably provide 2D variants for it. (👉 https://github.com/MovingBlocks/joml-ext/issues/14)